_includes/nav.html

```diff
@@ -1,6 +1,15 @@
 <nav role="navigation" aria-label="Main navigation">
   <ul class="navigation-list">
-    {%- assign pages_list = site.html_pages | sort:"nav_order" -%}
+    {%- assign ordered_pages_list = site.html_pages | where_exp:"item", "item.nav_order != nil" -%}
+    {%- assign unordered_pages_list = site.html_pages | where_exp:"item", "item.nav_order == nil" -%}
+    {%- if site.nav_sort == 'case_sensitive' -%}
```

Comment:
In connection with my request for a change in `_config.yml` regarding the default for `nav_sort`, replacing lines 5-11 by the following makes `nil` equivalent to `case_sensitive`:

```
    {%- if site.nav_sort == 'case_insensitive' -%}
      {%- assign sorted_ordered_pages_list = ordered_pages_list | sort_natural:"nav_order" -%}
      {%- assign sorted_unordered_pages_list = unordered_pages_list | sort_natural:"title" -%}
    {%- else -%}
      {%- assign sorted_ordered_pages_list = ordered_pages_list | sort:"nav_order" -%}
      {%- assign sorted_unordered_pages_list = unordered_pages_list | sort:"title" -%}
    {%- endif -%}
```